packages/core/src/secretStore/secretKeyGenerator.test.ts

```diff
@@ -17,135 +17,50 @@
  *
  */
 
-import {Decoder, Encoder} from 'bazinga64';
+import {EncryptedStore, createCustomEncryptedStore, createEncryptedStore} from './encryptedStore';
+import {generateSecretKey} from './secretKeyGenerator';
 
-import {generateSecretKey, CorruptedKeyError} from './secretKeyGenerator';
-
-const systemCryptos = {
-  v0: {
-    encrypt: async (value: Uint8Array) => {
-      return value;
-    },
-    decrypt: async (value: Uint8Array) => {
-      return value;
-    },
-    version: undefined,
-  },
-
-  v01: {
-    encrypt: async (value: Uint8Array) => {
-      return Encoder.toBase64(value).asBytes;
-    },
-    decrypt: async (value: Uint8Array) => {
-      const str = new TextDecoder().decode(value);
-      return Decoder.fromBase64(str).asBytes;
-    },
-    version: undefined,
-  },
-
-  v1: {
-    encrypt: async (value: string) => {
-      const encoder = new TextEncoder();
-      return encoder.encode(value);
-    },
-    decrypt: async (value: Uint8Array) => {
-      const decoder = new TextDecoder();
-      return decoder.decode(value);
-    },
-    version: 1,
-  },
```

Comment:
Those tests for the handling of different system crypto version has been moved to the webapp (PR coming soon). 
The core should not be aware of any system crypto (use for Electron).